### PR TITLE
Install docker compose during vagrant up and increase stage logging

### PR DESCRIPTION
**Pull request by jebbinSCL**
Install docker compose
Add logging for beginning and ending of stages.

### CODE REVIEW

**Comment by jebbinSCL** [comment]
Making a small update to the commands

File: vagrantScripts/backend-docker.sh [hunk 1]
```diff
@@ -17,3 +18,15 @@ sudo apt-get install -y docker-engine
 
 # Add vagrant to the docker group
 sudo usermod -aG docker vagrant
+
+# Install docker compose
+sudo sh -c "curl -L https://github.com/docker/compose/releases/download/1.7.1/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose"
+sudo sh -c "chmod +x /usr/local/bin/docker-compose"
+
+# setup bash completion for zsh
+su -c "mkdir -p ~/.zsh/completion" vagrant
```
Comment (pallenSL) [inline vagrantScripts/backend-docker.sh]: I'm not sure this is the correct location.  I don't have a ~/.zsh directory - everything is in ~/.oh-my-zsh

Comment (jebbinSCL) [inline vagrantScripts/backend-docker.sh]: Removed for now.

**Comment by pallenSL** [comment]
Your final echo commands return 0.  Always.
So the up command blunders on even on error.  :(

**Pull request merged**